### ISSUE TITLE
ConcurrentModificationException when using jetty-maven-plugin in Jetty 12

### ISSUE THREAD

**Issue by lachlan-roberts** [eclipse/jetty.project#11908]
**Jetty version(s)**
12.0.10

**Jetty Environment**
ee8, ee9, ee10

**Java version/vendor** `(use: java -version)`
OpenJDK Runtime Environment Temurin-21+35 (build 21+35-LTS)

**Description**

When using my project with the following configuration (this happens with any ee environment)
```
<plugin>
  <groupId>org.eclipse.jetty.ee8</groupId>
  <artifactId>jetty-ee8-maven-plugin</artifactId>
  <version>${jetty.version}</version>
  <configuration>
    <deployMode>EXTERNAL</deployMode>
    <jettyBase>${project.basedir}/jetty-base</jettyBase>
  </configuration>
</plugin>
```

I get the following exception:
```
java.util.ConcurrentModificationException
        at java.base/java.util.HashMap$HashIterator.nextNode(HashMap.java:1605)
        at java.base/java.util.HashMap$KeyIterator.next(HashMap.java:1628)
        at org.eclipse.jetty.start.Main.processCommandLine(Main.java:345)
        at org.eclipse.jetty.start.Main.main(Main.java:74)
```

the contents of `jetty-base` is:
```
jetty-base/
├── etc
│   ├── jetty-openid-baseloginservice.xml
│   └── realm.properties
├── resources
│   └── jetty-logging.properties
└── start.d
    ├── ee8-deploy.ini
    ├── ee8-openid.ini
    ├── ee8-webapp.ini
    ├── http.ini
    └── openid.ini
```

**Comment by janbartel** [comment]
Which goal are you using? How is the plug-in executing, is it at the command line or bound to a build phase?

**Comment by lachlan-roberts** [comment]
@janbartel I am using `mvn jetty:run` from the command line.

**Comment by janbartel** [comment]
Fixed in issue #11909 via #11913
